app/controllers/concerns/conferencable.rb

```diff
@@ -0,0 +1,18 @@
+module Conferencable
```

Comment:
We aren't super opinionated on syntax, but there's **this one thing™** we care about (see the [contribution guide](https://github.com/rails-girls-summer-of-code/rgsoc-teams/blob/master/CONTRIBUTING.md#write-code)):
>  indentation: two whitespaces

So, for the sake of consistency, would be super cool if you could update this to be with 2 instead of 4 spaces indentation 💚